src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+import java.util.Scanner;
+
+/**
+ * Interface that users interact with.
+ */
+public class Ui {
+    private final Scanner sc = new Scanner(System.in);
+
+    /**
+     * Prints greeting message when users start up the bot.
+     */
+    public void greet() {
+        System.out.println("______________________________________\n"
+                + "Hi, I'm Chatty\n"
+                + "What do you need to do today?\n"
+                + "______________________________________");
+    }
+
+    /**
+     * Displays a line for presentation.
+     */
+    public void showLine() {
+        System.out.println("______________________________________\n");
+    }
```

Comment:
Are the horizontal lines in 'greet' necessary given that this method exists?
For instance, could I do something similar to this?
```java
public void greet() {
    showLine();
    System.out.println("....");
    showLine();
}
```